drivers/hmis/app/graphql/schema.graphql

```diff
@@ -1300,6 +1305,11 @@ type FormItem {
   autofillValues: [AutofillValue!]
   bounds: [ValueBound!]
 
+  """
+  Label to use for placeholder and population table
+  """
+  briefText: String
```

Comment:
Select fields should have placeholders (`Select <x>...`). The `briefText` field gets used for that, if it exists.

It also gets used in the record picker dialog, so the table can show "Homeless on previous night" while the form asks "On the night before, did the client stay on the streets, ES or SH?"